techniques/aria/ARIA5.html

```diff
@@ -85,9 +85,6 @@ <h3>A slider</h3>
          <li>
                <a href="https://www.w3.org/TR/wai-aria/#states_and_properties">Accessible Rich Internet Applications (WAI-ARIA), Supported States and Properties</a>
             </li>
-         <li>
```

Comment:
is it intentional that the link was removed completely?

Reply:
@patrickhlauke 

> is it intentional that the link was removed completely?

Yes. There are already references to WAI-ARIA, and I think it is unnecessary to mention HTML-AAM in ARIA5.

Reply:
cool cool, just checking